### PR TITLE
Update install.sh

### PR DESCRIPTION
**Pull request by bindushree16**
https://github.com/Shippable/pm/issues/10666

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-ln -fs /usr/local/apache-maven-3.5.2/bin/mvn /usr/bin
-echo 'export PATH=$PATH:/usr/local/apache-maven-3.5.2/bin' >> /etc/drydock/.env
-
-echo "================ Installing apache-ant-1.10.1 ================="
-wget -nv https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.1-bin.tar.gz
-tar xzf apache-ant-1.10.1-bin.tar.gz -C /usr/local && rm -f apache-ant-1.10.1-bin.tar.gz
-ln -fs /usr/local/apache-ant-1.10.1/bin/ant /usr/bin
-echo 'export ANT_HOME=/usr/local/apache-ant-1.10.1' >> /etc/drydock/.env
-echo 'export PATH=$PATH:/usr/local/apache-ant-1.10.1/bin' >> /etc/drydock/.env
+wget -nv https://services.gradle.org/distributions/gradle-4.6-all.zip
```
Comment (ric03uec) [inline install.sh]: why dont we first export the version in a variable up top and use that variable thereafter. no point in repeating this multiple times

Comment (ric03uec) [reply install.sh]: something like https://github.com/dry-dock/u16/blob/master/install.sh#L89

Comment (bindushree16) [reply install.sh]: ok

**Comment by bindushree16** [comment]
there is no update in apache-ant version so i made it as 1.10.1 again

File: install.sh [hunk 2]
```diff
@@ -1,21 +1,30 @@
+
+GRADLE_VERSION=4.6
+echo "Installing gradle version: $GRADLE_VERSION"
 echo "================ Installing gradle ================="
-wget -nv https://services.gradle.org/distributions/gradle-4.2.1-all.zip
-unzip -qq gradle-4.2.1-all.zip -d /usr/local && rm -f gradle-4.2.1-all.zip
-ln -fs /usr/local/gradle-4.2.1/bin/gradle /usr/bin
-echo 'export PATH=$PATH:/usr/local/gradle-4.2.1/bin' >> /etc/drydock/.env
+wget -nv https://services.gradle.org/distributions/gradle-4.6-all.zip
```
Comment (ric03uec) [inline install.sh]: why is the GRADLE_VERSION not being used here?

**Pull request merged**